src/tests/results_tests/factories.py

```diff
@@ -59,13 +64,13 @@ class Meta:
     model_comment = ''
     batch_comment = ''
     config = {}
+    matrix_rel = factory.SubFactory(MatrixFactory)
     experiment_rel = factory.SubFactory(ExperimentFactory)
     train_end_time = factory.fuzzy.FuzzyNaiveDateTime(datetime(2008, 1, 1))
     test = False
-    train_matrix_uuid = factory.fuzzy.FuzzyText()
+    train_matrix_uuid = "efgh"
```

Comment:
This could get out of date with the Matrix's version of the id. I think you need to use the subfactory's version of the field.

http://factoryboy.readthedocs.io/en/latest/recipes.html#copying-fields-to-a-subfactory

I think this would be:

`train_matrix_uuid = factory.SelfAttribute('matrix_rel.matrix_uuid')`